Cannot resolve dependencies on new clone

I cloned this project and attempted to follow the installation instructions. I completed the following steps but compilation fails due to missing dependencies.

- Clone the repository
- Run `npm install`
- Run `npm run build-plugin`

```
npm run build-plugin

> vscode-java-test@0.37.1 build-plugin
> node scripts/buildJdtlsExt.js

[INFO] Scanning for projects...
[WARNING] No explicit target runtime environment configuration. Build is platform dependent.
[WARNING] No explicit target runtime environment configuration. Build is platform dependent.
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[INFO] Fetching p2.index from https://download.eclipse.org/releases/2022-09/202209141001/ (140B)
[INFO] Adding repository https://download.eclipse.org/releases/2022-09/202209141001
[INFO] Fetching p2.index from https://download.eclipse.org/jdtls/snapshots/repository/latest/ (172B)
[INFO] Adding repository https://download.eclipse.org/jdtls/snapshots/repository/latest
[INFO] Fetching p2.index from https://download.eclipse.org/tools/orbit/R-builds/R20200529191137/repository/ (134B)
[INFO] Adding repository https://download.eclipse.org/tools/orbit/R-builds/R20200529191137/repository
[INFO] Fetching p2.index from https://download.eclipse.org/tools/orbit/downloads/drops2/R20200529191137/repository/ (172B)
[INFO] Resolving dependencies of MavenProject: com.microsoft.java.test:com.microsoft.java.test.plugin:0.37.1 @ /Users/kevinsookocheff/Projects/vscode-java-test/java-extension/com.microsoft.java.test.plugin/pom.xml
[INFO] {osgi.os=macosx, osgi.ws=cocoa, org.eclipse.update.install.features=true, osgi.arch=aarch64}
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: com.microsoft.java.test.plugin 0.37.1
[ERROR]   Missing requirement: org.eclipse.jdt.ls.core 1.18.0.202211151951 requires 'osgi.bundle; org.eclipse.lsp4j [0.12.0,0.13.0)' but it could not be found
[ERROR]   Cannot satisfy dependency: com.microsoft.java.test.plugin 0.37.1 depends on: osgi.bundle; org.eclipse.jdt.ls.core 0.0.0
[ERROR]
[ERROR] See https://wiki.eclipse.org/Tycho/Dependency_Resolution_Troubleshooting for help.
[ERROR] Cannot resolve dependencies of MavenProject: com.microsoft.java.test:com.microsoft.java.test.plugin:0.37.1 @ /Users/kevinsookocheff/Projects/vscode-java-test/java-extension/com.microsoft.java.test.plugin/pom.xml: See log for details -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException
node:child_process:891
    throw err;
```

--- a/java-extension/com.microsoft.java.test.plugin/.classpath
+++ b/java-extension/com.microsoft.java.test.plugin/.classpath
@@ -1,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry kind="output" path="target/classes"/>
+</classpath>

--- a/java-extension/com.microsoft.java.test.plugin/.project
+++ b/java-extension/com.microsoft.java.test.plugin/.project
@@ -1,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>com.microsoft.java.test.plugin</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.pde.ManifestBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.pde.SchemaBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.m2e.core.maven2Nature</nature>
+		<nature>org.eclipse.pde.PluginNature</nature>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+	</natures>
+	<filteredResources>
+		<filter>
+			<id>1668558612463</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>

--- a/java-extension/com.microsoft.java.test.plugin/.settings/org.eclipse.core.resources.prefs
+++ b/java-extension/com.microsoft.java.test.plugin/.settings/org.eclipse.core.resources.prefs
@@ -1,0 +1,2 @@
+eclipse.preferences.version=1
+encoding/<project>=UTF-8

--- a/java-extension/com.microsoft.java.test.plugin/.settings/org.eclipse.jdt.core.prefs
+++ b/java-extension/com.microsoft.java.test.plugin/.settings/org.eclipse.jdt.core.prefs
@@ -1,0 +1,8 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.compliance=1.8
+org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
+org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+org.eclipse.jdt.core.compiler.release=disabled
+org.eclipse.jdt.core.compiler.source=1.8

--- a/java-extension/com.microsoft.java.test.plugin/.settings/org.eclipse.m2e.core.prefs
+++ b/java-extension/com.microsoft.java.test.plugin/.settings/org.eclipse.m2e.core.prefs
@@ -1,0 +1,4 @@
+activeProfiles=
+eclipse.preferences.version=1
+resolveWorkspaceProjects=true
+version=1

--- a/java-extension/com.microsoft.java.test.plugin/target.target
+++ b/java-extension/com.microsoft.java.test.plugin/target.target
@@ -8,7 +8,7 @@
             <unit id="org.eclipse.equinox.sdk.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.jdt.source.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
-            <repository location="https://download.eclipse.org/eclipse/updates/4.25/"/>
+            <repository location="https://download.eclipse.org/eclipse/updates/4.26-I-builds/I20221027-1800"/>
         </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
             <unit id="org.eclipse.xtend.sdk.feature.group" version="0.0.0"/>

--- a/java-extension/pom.xml
+++ b/java-extension/pom.xml
@@ -74,5 +74,11 @@
             <layout>p2</layout>
             <url>https://download.eclipse.org/tools/orbit/R-builds/R20200529191137/repository/</url>
         </repository>
+        <!-- only used to download lsp4j 0.12.0 and its dependencies -->
+        <repository>
+            <id>202112</id>
+            <layout>p2</layout>
+            <url>https://download.eclipse.org/releases/2021-12/202112081000/</url>
+        </repository>
     </repositories>
 </project>

--- a/package.json
+++ b/package.json
@@ -70,7 +70,7 @@
             "./server/org.apiguardian.api_1.1.2.jar",
             "./server/org.eclipse.jdt.junit4.runtime_1.3.0.v20220609-1843.jar",
             "./server/org.eclipse.jdt.junit5.runtime_1.1.0.v20220715-1030.jar",
-            "./server/org.opentest4j_1.2.0.v20190826-0900.jar",
+            "./server/org.opentest4j_1.2.0.v20211018-1956.jar",
             "./server/com.microsoft.java.test.plugin-0.37.1.jar"
         ],
         "viewsWelcome": [

